### PR TITLE
Morse

### PR DESCRIPTION
**Pull request by mutinhiri**
In this Pr we implemented Ruby method to decode a morse message

### CODE REVIEW

File: morse.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z', '.----' => '1', '..---' => '2', '...--' => '3', '....-' => '4', '.....' => '5',
+    '-....' => '6', '--...' => '7', '---..' => '8', '----.' => '9', '-----' => '0'
+  }
+  @result = @morse_to_char[char]
+  if @result.nil?
+    ' '
+  else
+    @result
+  end
```
Comment (uwadonat) [inline morse.rb:14]: - [ ] Let's add a method to display at least one decoded character.

File: morse.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+end
+
+def decoded_word(word)
+  word = word.split
+  result = ''
+  word.each do |char|
+    result += decoded_char(char)
+  end
+  result
+end
```
Comment (uwadonat) [inline morse.rb:28]: - [ ] Let's add a method to display at least one decoded word.

File: morse.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+def decode(message)
+  message = message.split('   ')
+  result = []
+  message.each do |char|
+    result.push(decoded_word(char))
+  end
+  result.join(' ')
+end
```
Comment (uwadonat) [inline morse.rb:37]: - [ ] Let's add a method to display at least two words decoded.

File: morse.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+puts decode('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-. / .-. ..- -... .. . ...')
```
Comment (uwadonat) [inline morse.rb:41]: - [ ] make sure your final output shows 4 outputs including the one you are displaying here.

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **@Team,** :hand: 

Good job so far! :mechanical_arm:
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight! :+1: 
- Your commit messages are very descriptive ✔️


## Status: Required Changes ♻️
**_Check the comments under the review._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by omar-labana** [comment]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**